less/forum/Composer.less

```diff
@@ -10,94 +10,107 @@
     cursor: pointer;
     background: @control-bg;
   }
-}
-.Composer-controls {
-  list-style: none;
-  padding: 0;
-  margin: 0;
-}
-.ComposerBody-content {
-  .minimized & {
-    white-space: nowrap;
-    overflow: hidden;
-    text-overflow: ellipsis;
+
+  &-controls {
+    position: absolute;
+    right: 10px;
+    top: 10px;
+    z-index: 1;
+    list-style: none;
+    padding: 0;
+    margin: 0;
+
+    & li {
+      display: inline-block;
+    }
+    .minimized & {
+      top: 7px;
+    }
+    .fa-minus.minimize {
+      vertical-align: -5px;
+    }
   }
 }
-.ComposerBody-header {
-  list-style: none;
-  padding: 1px 0;
-  margin: 0 0 10px;
 
-  .minimized & {
-    pointer-events: none;
+.ComposerBody {
+  &-content {
+    .minimized & {
+      white-space: nowrap;
+      overflow: hidden;
+      text-overflow: ellipsis;
+    }
   }
 
-  > li {
-    display: inline-block;
-    margin-right: -4px;
-  }
-  h3 {
-    margin: 0;
-    line-height: 1.5em;
-    color: @secondary-color;
+  &-header {
+    list-style: none;
+    padding: 1px 0;
+    margin: 0 0 10px;
 
-    &, input, a {
-      font-size: 14px;
-      font-weight: normal;
+    .minimized & {
+      pointer-events: none;
     }
-    input, a {
-      color: inherit;
+
+    > li {
+      display: inline-block;
+      margin-right: -4px;
     }
-    input {
-      font-size: 16px;
-      width: 500px;
+    h3 {
+      margin: 0;
+      line-height: 1.5em;
+      color: @secondary-color;
 
-      &, &[disabled], &:focus {
-        background: none;
-        border: 0;
-        padding: 0 20px 0 0;
-        height: auto;
+      &,
+      input,
+      a {
+        font-size: 14px;
+        font-weight: normal;
+      }
+
+      input,
+      a {
+        color: inherit;
+      }
+
+      input {
+        font-size: 16px;
+        width: 500px;
+
+        &,
+        &[disabled],
+        &:focus {
+          background: none;
+          border: 0;
+          padding: 0 20px 0 0;
+          height: auto;
+        }
+      }
+
+      .icon {
+        font-size: 14px;
+        margin-right: 3px;
       }
-    }
-    .icon {
-      font-size: 14px;
-      margin-right: 3px;
     }
   }
-}
-.fa-minus.minimize {
-  vertical-align: -5px;
-}
-.Composer-controls {
-  position: absolute;
-  right: 10px;
-  top: 10px;
-  z-index: 1;
-
-  & li {
-    display: inline-block;
-  }
-  .minimized & {
-    top: 7px;
-  }
-}
-.ComposerBody-loading {
-  position: absolute;
-  top: 0;
-  left: 0;
-  right: 0;
-  bottom: 0;
-  background: fade(@body-bg, 90%);
-  display: none;
-  border-radius: @border-radius @border-radius 0 0;
-
-  &.active {
-    display: block;
+
+  &-loading {
```

Comment:
Could we revert some of the stylistic changes? It's making it a bit difficult to see what has and hasn't changed